blog.html

```diff
@@ -22,6 +22,146 @@
+                    </li>
+                    <li><b><a
+                            href="https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=195726330">KIP-797</a>:
+                        Accept duplicate listener on port for IPv4/IPv6: </b><br>Until now, Kafka has not supported
+                        duplicate listeners on the same port. This works when using only a single IP stack, but presents
+                        an issue if you are working with both IPv4 and IPv6. With KIP-797, brokers can be configured
+                        with listeners that have the same port on different IP stacks. This update does not affect
+                        advertised listeners, which already have this feature.
+                    </li>
+                    <li><b><a
+                            href="https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=225152035">KIP-863</a>:
+                        Reduce CompletedFetch#parseRecord() memory copy: </b><br>reduces memory allocation and improves
+                        memory performance during record deserialization by using a ByteBuffer instead of byte[] for
+                        deserialization, which improves efficiency. Updated public interfaces include the Deserializer
+                        class, ByteBufferDeserializer class, and StringDeserializer class.
+                    </li>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-868+Metadata+Transactions">KIP-868</a>:
+                        Metadata Transactions: </b><br> improves the overall durability of the KRaft layer by adding
+                        metadata transactions that consist of:
+                        <ul>
+                            <li>ssBeginTransaction</li>
+                            <li>Number of records</li>
+                            <li>EndTransaction or AbortTransaction</li>
+                        </ul>
+                        KRaft uses record batches as a mechanism for atomicity. Typically, there was a limit to the
+                        fetch size on the Raft consensus layer, and the controller could generate a set of atomic
+                        records that exceeded this limit. This update introduces marker records that allow larger sets
+                        of atomic records to be sent to the Raft consensus layer in multiple batches. This bypasses the
+                        fetch limit.
+                    </li>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-902%3A+Upgrade+Zookeeper+to+3.8.2">KIP-902</a>:
+                        Upgrade Zookeeper to 3.8.2:</b>upgrades the ZooKeeper version that is bundled with Kafka to
+                        version 3.8.2. The new version includes several updates and security improvements. This is the
+                        last planned update for ZooKeeper in the 3.8.x releases, as it will be removed in Apache Kafka
```

Comment:
Should `3.8.x releases` be `3.6.x releases`? As I think you mean it's the last time Kafka updates the ZooKeeper dependency. I'm actually not sure we want to say that. For example we may need to bump ZooKeeper's version if a CVE is found.

Reply:
It meant the last time to update but any critical fixes or CVE updates will be honoured. Removed this sentence as suggested to avoid any confusion.